acl/types.go

```diff
@@ -51,3 +57,34 @@ const (
 	// EACLPublicAppendName is a well-known name for 0x0FBF9FFF basic ACL.
 	EACLPublicAppendName = "eacl-public-append"
 )
+
+// ParseBasicACL parse string ACL (well-known names or hex representation).
+func ParseBasicACL(basicACL string) (uint32, error) {
+	switch basicACL {
+	case PublicBasicName:
+		return PublicBasicRule, nil
+	case PrivateBasicName:
+		return PrivateBasicRule, nil
+	case ReadOnlyBasicName:
+		return ReadOnlyBasicRule, nil
+	case PublicAppendName:
+		return PublicAppendRule, nil
+	case EACLPublicBasicName:
+		return EACLPublicBasicRule, nil
+	case EACLPrivateBasicName:
+		return EACLPrivateBasicRule, nil
+	case EACLReadOnlyBasicName:
+		return EACLReadOnlyBasicRule, nil
+	case EACLPublicAppendName:
+		return EACLPublicAppendRule, nil
+	default:
+		basicACL = strings.Trim(strings.ToLower(basicACL), "0x")
```

Comment:
While `Trim` will do the job, it will also cut e.g. `0xxx0x`. I believe, `TrimPrefix` was meant.